images/builders/mlflow/mlflow/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is there any reason to use `bash` now? You did not change the code, so it does not seem to use any bash-like features...

Reply:
I am not really sure how it worked before, maybe the previous image loads bash by default, but the reason for using bash is because conda is configured to init on bash (sh is not even supported by conda).

Reply:
Ha